### ISSUE TITLE
Model Training Logs

### ISSUE THREAD

**Issue by shreyasinghal-17** [shivammehta25/Matcha-TTS#18]
Attaching my loss card from tensorboard/ does everything looks okay?  val_epoch seems to have converged, what were the final losses you were getting. Any other thing I need to keep an eye on?

![image_2023-10-04_22-12-21](https://github.com/shivammehta25/Matcha-TTS/assets/70097551/b909ad37-5af9-47bc-afea-7c7c2831b21b)

with smoothing = 0.6

**Comment by shivammehta25** [comment]
More details about individual losses can be seen in `sub_loss` card. The flow matching loss is noisy and is not the best representation of model performance (similar to diffusion loss). Further, from what I've seen it is okay for the duration predictor to overfit for a while mainly because of the limited val set (my val set was only 100 samples). 

Generally, training for longer helps, as these losses are not the best representative of the model's performance, similar to diffusion-type models.

**Comment by shreyasinghal-17** [comment]
I am getting good quality speech! I havent trained a vocoder yet, but with griffin lim it seems acceptable, I'm sure neural vocoders would yield great output.

Since I am new with this I have some questions, Should I ask them here or is there a better channel for discussion?

**Comment by shivammehta25** [comment]
> I'm sure neural vocoders would yield great output.

If you do not want to train a vocoder you can use the [universal HiFiGAN](https://drive.google.com/drive/folders/1YuOoV3lO2-Hhn1F2HJ2aQ4S0LC1JdKLd). 

> Since I am new to this I have some questions, Should I ask them here or is there a better channel for discussion?

You can continue posting them here, or reach out to me by email `smehta`(at)`kth`(dot)`se` . Additionally, I can also answer them on [LinkedIn](https://www.linkedin.com/in/shivammehta25/).

**Comment by shreyasinghal-17** [comment]
I tried using the universal vocoder but that dint went well, I'll figure it out tho. 

Thanks I'll close this issue then and reach out on other channels.

**Comment by p0p4k** [comment]
@shreyasinghal-17 Hi, can you share your dur_loss graphs/logs? Thanks

**Comment by shreyasinghal-17** [comment]
![dur_loss2](https://github.com/shivammehta25/Matcha-TTS/assets/70097551/fc99fe71-ddda-4a49-a8c3-9047c28c5d2b)
![dur_loss1](https://github.com/shivammehta25/Matcha-TTS/assets/70097551/c13458c3-7e09-402b-aaed-153ab83a809c)

@p0p4k  I trained this for more iterations but for some reason tensorboard is not showing the comeplete data points

**Comment by aizhiqi-work** [comment]
@shreyasinghal-17 Hi, can you share your diff_loss&prior graphs/logs? Thanks

**Comment by ummjevel** [comment]
hi, 

Did you find why tensorboard is not showing the complete data points? I ran into the same issue. 

Thanks.

**Comment by p0p4k** [comment]
@ummjevel what do you mean by complete data points?

**Comment by ummjevel** [comment]
Hi, @p0p4k
First, thank you for asking.
I mean i set the max_epochs 2000, but tensorboard logs are not saved until 2000 epochs. 
as you see the upper image, the tensorboard log wrote until 50k but the maximum step is 782k.
(ljspeech train data : 12,500, batch size : 32, 12500/32 = 390, 390*2000 = 782,000)
as you see the lower image, i can find the saved ckpt files so training and saving checkpoints per steps/epoch is ok.

![image](https://github.com/user-attachments/assets/721be507-c21e-430e-bc26-908b53b84129)
![image](https://github.com/user-attachments/assets/65b70ff9-2769-4300-90db-32f209eddad7)

is there a config that i miss?
